### ISSUE TITLE
When remotely fetching followers enabled, follower count doesn't update accordingly to remote data.

### ISSUE THREAD

**Issue by d4f5409d** [sk22/megalodon#632]
**Describe the bug**

When the remote data gives 100 followers for example, Megalodon pulls the follower count data from the local instance, which is around 34. And it looks odd when there is this cool feature that can fetch remote data.

**To reproduce**

I think it's obvious

**Does this happen in the official app?**

Does this issue also occur with the respective upstream release?
(Please test using the respective `upstream-xxxxxx.apk` provided in [Releases](https://github.com/sk22/megalodon/releases) or at least using the current Mastodon version from the Play Store)

> No

> In case it does, please consider filing an [upstream bug report](https://github.com/mastodon/mastodon-android/issues) instead.
> If this bug is seriously impacting your usage or you think I might want to try to fix it for Megalodon, feel free to still create this issue!

**Screenshots and screen recordings**

If applicable, add screenshots (and screen recordings, if possible) to help explain your problem.

**Version**

Megalodon version: the latest

**Crash log**

If you know your way around Android development tools, please consider attaching a crash log, if possible.

**Comment by sk22** [comment]
> I think it's obvious

it's not - what count do you think should be updated?
the thing is that the remote listing is only loaded when opening the follower/fav/boost lists (so i don't have to make two requests for every post). like, yeah, i could update the numbers elsewhere, but once refreshing them, they'd be back down at the local numbers again.
so it's kind of intentional that you only get to see the real numbers when opening the list. i am, however, considering automatically fetching profiles (maybe even opened posts?) from the remote instance

**Comment by d4f5409d** [comment]
Yeah, that would be cool! Remote fetching profiles, favorites and boosts (counts) would be really nice. Thank you.

I meant by the way when you open the profile ot shows a follower count next to the display name, right? I meant that one.
